src/software/ai/hl/stp/play/halt_play_test.cpp

```diff
@@ -31,7 +31,7 @@ TEST(StopPlayTest, test_stop_play_returns_correct_tactics)
     {
         try
         {
-            dynamic_cast<StopTactic*>(t.get());
+            auto temp = dynamic_cast<StopTactic*>(t.get());
```

Comment:
could we rename to `unused` just to be super clear?

Reply:
Done